aws-api/src/test/java/com/amplifyframework/api/aws/GsonGraphQLResponseFactoryTest.java

```diff
@@ -122,15 +123,59 @@ public void partialResponseRendersWithTodoDataAndErrors() throws ApiException {
         // Assert that we parsed the errors successfully.
         assertNotNull(response.getErrors());
 
-        final List<GraphQLResponse.Error> expectedErrors = Arrays.asList(
-            new GraphQLResponse.Error("failed"),
-            new GraphQLResponse.Error("failed"),
-            new GraphQLResponse.Error("failed")
-        );
+        for (GraphQLResponse.Error error : response.getErrors()) {
+            assertEquals(error.getMessage(), "failed");
```

Comment:
These are backwards, the meaning of the arguments are `assertEquals(expectedValue, actualValue)` -- `"failed"` is the one you expected to find, `error.getMessage()` is the actual value under test.

Reply:
Fixed!